packages/components/skeleton/src/skeleton.ts

```diff
@@ -5,13 +5,22 @@ import styles from './skeleton.scss.js';
 
 export type SkeletonEffect = 'none' | 'shimmer' | 'pulse' | 'sheen';
 
+export type SkeletonVariant = 'circle' | 'default';
+
 export class Skeleton extends LitElement {
   /** @private */
   static override styles: CSSResultGroup = styles;
 
-  /** Skeleton's effect. */
+  /** Skeleton's effect.
+   *  @type {'none' | 'shimmer' | 'pulse' | 'sheen'}
```

Comment:
I'm always wondering if the cem analyser needs this, or if it can read typescript and figure out the types from there?

Reply:
Maybe we can test this plugin in a separated task?

Reply:
New task created: https://github.com/sl-design-system/components/issues/832